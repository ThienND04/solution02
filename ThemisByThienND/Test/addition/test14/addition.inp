9
2934 9639 1301 943658
1392 9778 1934 850878
3175 10011 1384 938165
2219 5690 2049 522503
2199 8929 3144 542612
1940 9126 3117 597157
2532 7787 2851 780278
1213 9158 2495 807826
3034 7034 1522 877633
