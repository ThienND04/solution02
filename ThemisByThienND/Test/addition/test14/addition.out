15
29
13
7
4
4
6
12
0
